kevm-pyk/src/kevm_pyk/kproj/evm-semantics/optimizations.md

```diff
@@ -81,7 +88,8 @@ module EVM-OPTIMIZATIONS
       ...
     </kevm>
     requires ( #if USEGAS #then Gverylow < SCHED > <=Gas GAVAIL #else true #fi )
-     andBool ( #sizeWordStack( WS ) <=Int 1023 )
+     andBool ( 0 <=Int WSSize )
+     andBool ( WSSize <=Int 1023 )
```

Comment:
Do we need a `#rangeWSSize`? I see these constraints are only used in `optimizations.md`.

Reply:
No, we can't, it's a bit dynamic.